mlir/include/mlir/Dialect/LLVMIR/LLVMLinkerInterface.h

```diff
@@ -18,6 +22,12 @@ class LLVMSymbolLinkerInterface
   static unsigned getBitWidth(Operation *op);
   static UnnamedAddr getUnnamedAddr(Operation *op);
   static void setUnnamedAddr(Operation *op, UnnamedAddr val);
+  static std::optional<uint64_t> getAlignment(Operation *op);
+  static bool isConstant(Operation *op);
+  static llvm::StringRef getSection(Operation *op);
+  static uint32_t getAddressSpace(Operation *op);
```

Comment:
Could some of these be const?

Reply:
The methods are already static. Do you mean the arguments? MLIR unfortunately does not support using const on core IR types (https://mlir.llvm.org/docs/Rationale/UsageOfConst/)

Reply:
Oh crap! Sorry, read too fast...